utils/mediaupload.js

```diff
@@ -115,14 +115,18 @@ export function mediaUpload( {
 
 		return createMediaFromFile( mediaFile, additionalData ).then(
 			( savedMedia ) => {
-				const mediaObject = {
+				let mediaObject = {
```

Comment:
I think we can still use const, we are not changing the mediaObject reference we are just changing a key in the object.

Reply:
> I think we can still use const, we are not changing the mediaObject reference we are just changing a key in the object.

This is bizarre and seems like a bug. Constant means _constant_, or unchanging. Reassignment, even to some subset of the data, isn't permitted in PHP.

I've made the change regardless 4f3acb7f0